agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2367,6 +2368,33 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
         }
     }
 
+    for (const auto &neighbor_device : m_1905_neighbor_devices) {
+        auto tlv1905NeighborDevice = cmdu_tx.addClass<ieee1905_1::tlv1905NeighborDevice>();
```

Comment:
This is not correct: there should be a tlv1905NeighborDevice *per local interface*. Since we only model 1 local interface (the bridge), the should be only one tlv1905NeighborDevice. So something like:
```
        auto tlv1905NeighborDevice = cmdu_tx.addClass<ieee1905_1::tlv1905NeighborDevice>();
        tlv1905NeighborDevice->mac_local_iface() = network_utils::mac_from_string(bridge_info.mac);
        if (!tlv1905NeighborDevice->alloc_mac_al_1905_device(m_1905_neighbor_devices.size()))
            ...
        size_t index = 0;
        for (const auto &neighbor_device : m_1905_neighbor_devices) {
            auto mac_al_1905_device_tuple = tlv1905NeighborDevice->mac_al_1905_device(index);
            ...
            index++;
        }
```

Reply:
Yes, you are right.
fixed it.